src/index.html

```diff
@@ -2,12 +2,503 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
+    <title>BIKE</title>
     <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <header class="header">
+      <div class="container">
+        <div class="top-action">
+          <div class="logo"></div>
+          <a
+            href="tel: +1 234 5555-55-55"
+            class="icon icon--phone"
+          >
+          </a>
+          <a href="#menu" class="icon icon--menu"></a>
+        </div>
+
+        <h1 class="header__title">Take the Streets</h1>
+      </div>
+    </header>
+
+    <nav class="page__menu menu" id="menu">
```

Comment:
Menu should stretch to the full height of the page
![image](https://user-images.githubusercontent.com/37161315/170644891-3ed6352b-4e75-415a-8170-f3a2d785e739.png)

Reply:
Also, checklist 18th item